src/models/goal.js

```diff
@@ -0,0 +1,22 @@
+const {
+  Model,
+} = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Goal extends Model {
+    static associate(models) {
+      Goal.belongsToMany(models.Topic, { through: models.TopicGoal, foreignKey: 'goalId', as: 'topics' });
+      // Goal.hasMany(models.Ttaplan, { foreignKey: 'goalId', as: 'ttaplans' });
```

Comment:
Why is this commented out?

Reply:
In the current design, having a many to many relationship would need "belongsToMany" associations. That's what sequelize expects.